oonib/tests/test_bouncer.py

```diff
@@ -72,6 +72,22 @@
     test-helper: {fake_test_helper: 'fake_hostname'}
 """
 
+production_bouncer_file = """
+collector:
+  httpo://ihiderha53f36lsd.onion:
+    collector-alternate:
+    - {address: 'https://a.collector.ooni.io:4441', type: https}
```

Comment:
Actually the canonical bouncer is currently using this address. Though anyways what is in here doesn't really matter for the purpose of the unittest.